### ISSUE TITLE
Add possibility to use env vars

### ISSUE THREAD

**Issue by vvval** [spiral/roadrunner#142]
It would be very helpful if `.rr.yaml` could support environment variables 
```yaml
jobs:
  amqp:
    addr: ${AMQP_CONNECTION}
```
or using several config files at once.
```bash
rr serve -c .rr.yaml -c .vulnerable.rr.yaml
```

The case is to have vulnerable credentials outside of the GIT repository

**Comment by wolfy-j** [comment]
Coming in 1.3.8

**Comment by wolfy-j** [comment]
Available in 1.4.0
